src/components/sidebar/helpers/generate-resources-nav-items.ts

```diff
@@ -109,34 +123,45 @@ function getCertificationsLink(productSlug?: ProductSlug): {
  * Generates the sidebar nav items for the Resources section of the sidebar.
  * Optionally accepts a Product slug for customization of links.
  */
-const generateResourcesNavItems = (productSlug?: ProductSlug) => {
+function generateResourcesNavItems(
+	productSlug?: ProductSlug
+): ResourceNavItem[] {
 	const additionalResources = generateAdditionalResources(productSlug)
 
-	return [
-		{ heading: 'Resources' },
-		{
+	const resourceNavItems: ResourceNavItem[] = [{ heading: 'Resources' }]
+
+	// Add a "Tutorials" link for all products except Sentinel
+	if (productSlug !== 'sentinel') {
+		resourceNavItems.push({
 			title: 'Tutorial Library',
 			href: getTutorialLibraryUrl(productSlug),
-		},
-		...getCertificationsLink(productSlug),
-		{
-			title: 'Community Forum',
-			href: productSlug
-				? COMMUNITY_LINKS_BY_PRODUCT[productSlug]
-				: DEFAULT_COMMUNITY_FORUM_LINK,
-		},
-		{
-			title: 'Support',
-			href: DEFAULT_SUPPORT_LINK,
-		},
-		{
-			title: 'GitHub',
-			href: productSlug
-				? GITHUB_LINKS_BY_PRODUCT_SLUG[productSlug]
-				: DEFAULT_GITHUB_LINK,
-		},
-		...additionalResources,
-	]
+		})
+	}
+
+	resourceNavItems.push(
+		...[
+			...getCertificationsLink(productSlug),
+			{
+				title: 'Community Forum',
+				href: productSlug
+					? COMMUNITY_LINKS_BY_PRODUCT[productSlug]
+					: DEFAULT_COMMUNITY_FORUM_LINK,
+			},
+			{
+				title: 'Support',
+				href: DEFAULT_SUPPORT_LINK,
+			},
+			{
+				title: 'GitHub',
+				href: productSlug
+					? GITHUB_LINKS_BY_PRODUCT_SLUG[productSlug]
+					: DEFAULT_GITHUB_LINK,
+			},
+			...additionalResources,
+		]
+	)
+
+	return resourceNavItems
```

Comment:
What do you think about changing it something like this:
```
		...( productSlug !== 'sentinel' ? { // Add a "Tutorials" link for all products except Sentinel
			title: 'Tutorial Library',
			href: getTutorialLibraryUrl(productSlug),
		} : {}),
```
This way the LOC aren't increased as much, and it's still readable/achieves the same effect.